rig-postgres/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "rig-postgres"
+version = "0.1.0"
+edition = "2021"
+description = "PostgreSQL-based vector store implementation for the rig framework"
+license = "MIT"
```

Comment:
Need to add `readme` and `repository` fields or else crates.io complains

Reply:
updated